### ISSUE TITLE
Retraining the lower layers of the inception model for a better classifier. 

### ISSUE THREAD

**Issue by scm-ns** [tensorflow/tensorflow#14235]
https://github.com/tensorflow/tensorflow/blob/4e75ae1f1e8c6479cfa86fde1a940453945e6671/tensorflow/examples/image_retraining/retrain.py#L26

Hi, I am trying to use the inception model to learn to classify between about 200 different object types.  I tried the approach mentioned here, which seems to only retrain the last few layer of the model, and I can't get about ~ 60% accuracy even when I am training the model. I imagine if I pass on the gradient and change the weights in the lower layers, that I will be able to improve the accuracy of the model when applied to my data set. 

How do I go about doing this ? Any guidance on this being done else where would be extremely helpful.

**Comment by asimshankar** [comment]
The `retrain.py` example adds a new layer to the end of an existing model. Whether a single layer is sufficient, or whether the number of parameters chosen is sufficient, or how much data you should have to train effectively, or what the most effective way to train your classifier of 200 objects is are broad questions which aren't a matter of a TensorFlow bug or feature request :)

Apologies, but I'm going to close this issue out since the TensorFlow github issues are meant for bugs and feature requests.

**Comment by scm-ns** [comment]
I wasn't asking a broad question of creating my own model. I was asking how can I pass the gradients down to the lower layers of the Inception Model that is provided by tensorflow. Only the last layers is retrained, how can I retrain the lower layers of an existing model ? 

I apologize if I was not clear earlier.
